app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -67,7 +67,9 @@ const processVMTasks = vmTasks => {
       options: {}
     };
 
-    if (task.options.playbooks) {
+    const hasPlaybookService = task.options.playbooks;
+
+    if (hasPlaybookService) {
```

Comment:
We removed the unnecessary const declaration and moved `task.options.playbooks` directly into the conditional here [(review comment)](https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/442#discussion_r204044128)